wiki/People/Tournament_Committee/de.md

```diff
@@ -1,7 +1,7 @@
 ---
 tags:
   - TC
-outdated_since: 9661014ab900317b13040f5f3a2d36ce6b154b07
+outdated_since: 79edb3a3685e1e9f35c34c9733b2cfbff5eb2752
```

Comment:
don't change outdated_since if it's already there
you can think of these as saying the article is "outdated since the changes to EN at \<time\>", new changes to EN don't make the article any less outdated.

Reply:
That makes a lot of sense that 3:48 AM me would probably not have properly understood.

Ran the CI script and it threw the outdated error. Will roll back.